features/step_definitions/web_steps.rb

```diff
@@ -183,6 +183,11 @@ def with_scope(locator)
   end
 end
 
+Then(/^I should see "([^"]*)" or "([^"]*)"$/) do |text1, text2|
```

Comment:
I might be missing it -- I don't entirely trust Ctrl+F on pull requests -- but I don't think this is used.

Reply:
It's definitely unused: https://github.com/elzj/otwarchive/commit/75b037d6b4222ee5d30e456d4ecb6a1aceabec6a#diff-0fa40971eef4a01211c40236d846f289. It was added to deal with the PDT/PST time zone name issue in that one challenge test.